docs/project_wheres_religion_mobile/about.md

```diff
@@ -14,14 +14,17 @@ custom_edit_url: https://github.com/oss-slu/lrda_mobile
 
 - **Source Code:** [https://github.com/oss-slu/lrda_mobile](https://github.com/oss-slu/lrda_mobile) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/lrda_mobile)
 - **Client** Dr. Adam Park
-- **Current Tech Lead:** [Yash Bhatia](https://yashb196.github.io/yashb196/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashb196) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashbhatia238/)
```

Comment:
I think we should consider moving @yashb196 to a named role like "Senior Developer" or whatever Yash suggests either instead of or in addition to moving him to the full Developers: list. That would look something like:

```
**Client:** Dr. Adam Park
**Senior Developer:** [Yash Bhatia]...
**Current Tech Lead:** [Karthik Manginene]...
```

Reply:
sure daniel, I will contact Yash regarding this.

Reply:
I completed the task.
![Screenshot 2024-08-29 135625](https://github.com/user-attachments/assets/3c4353b2-3fb0-4308-a5be-1a6f8948477d)

Reply:
@yashb196 , do you have any preferences here?